manual/googlechrome-extensions/googlechrome-github-expandinizr/googlechrome-github-expandinizr.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-github-expandinizr</id>
+    <version>1.0.0</version>
```

Comment:
I think it's a good idea to use the current release of the extension as it appears current at time of package release. As a user if I see v1.0, first thing I do is click to see if there's a newer version out. IMHO v1.8.0 gives it more credibility and then a note that it will install the latest version whatever it is. Just my HO and how I package my Chrome extensions. :)